io/net/irqbalance.py.data/README.txt

```diff
@@ -0,0 +1,42 @@
+description:
+-------------
+This Program to assign and check different smp_affinity_list to IRQ of given IO device, and to caputure calltraces, dmesg stacks during the operations with validation of assigned values.
+Along with above operations with Program also covers following tests.
+
+    1. Setting up diffrent avialble CPU'S to IO based process by taskset and validating values set.
+    2. Making off/on [ offline/online ] of CPU's from min available CPU's to Max available CPU's serial fashion.
+    3. Setting diffent SMT levels and off/on using ppc64_cpu utils.
+
+
+-----------------------------
+Inputs Needed To Run Tests:
+-----------------------------
+
+1. For Network based test
+-------------------------
+peerip ---> IP of the Peer interface to be tested
+host_ip  --->   Specify host-IP for ip configuration.
+netmask  --->   specify netmask for ip configuration.
+interface --->  host interface through which we get host_ip
+ping_count ---> specity ping count for " ping flood" test, default set to "10000"
+
+2. For Storage based test:
+------------------------
+disk ----> Specify the disk path to run the test.
```

Comment:
@FarooqAbdulla01 one last comments,
please add the example here, in which formate the disk paths should be passed like
for nvme, /dev/nvme0 (controller), nvme0 or /dev/nvme0n1 (namespace)
for fc/vfc, sda, sdb or /dev/sda, /dev/sdb.
it will be easy for new users if we add this details.
similar for other inputs parameter. 
if the input values are straight forward like ip, interface, count etc, it is not required this details

Reply:
@Naresh-ibm Updated README.txt with example